python/podio/generator_utils.py

```diff
@@ -71,9 +71,10 @@ def _is_fixed_width_type(type_name):
 class DataType:
   """Simple class to hold information about a datatype or component that is
   defined in the datamodel."""
-  def __init__(self, klass):
+  def __init__(self, klass, schema_version=None):
     self.full_type = klass
     self.namespace, self.bare_type = _get_namespace_class(self.full_type)
+    self.schema_version = schema_version
```

Comment:
The roundtrip tests trip over this line here. The dumped yaml files will have `schema_version` as key in the yaml file, while the original one has `schemaversion`.

Either changing this to `schemaversion` or the yaml key to `schema_version` fixes the roundtrip tests. I would have a slight preference for renaming the yaml key to `schema_version` (mainly because it would mean less code changes).

Reply:
I actually already went for schema_version. good that we agreed there :)